conda_build/build.py

```diff
@@ -1026,6 +1026,16 @@ def tests_failed(m, move_broken, broken_dir, config):
 
 def check_external():
     if sys.platform.startswith('linux'):
+        ldd = external.find_executable('ldd')
+        if ldd is None:
+            sys.exit("""\
+Error:
+    Did not find 'ldd' in: %s
+    'ldd' is necessary for building conda packages on Linux with
+    relocatable ELF libraries.  Please install with your system package
```

Comment:
There are also implementations of `ldd` for [FreeBSD]( https://www.freebsd.org/cgi/man.cgi?query=ldd&sektion=1&manpath=FreeBSD+5.0-RELEASE ), [OpenBSD]( http://man.openbsd.org/OpenBSD-current/man1/ldd.1 ), [DragonFlyBSD]( https://www.dragonflybsd.org/cgi/web-man?command=ldd&section=ANY ), and most other Unix FWICT.